### ISSUE TITLE
2020/03 - MLConference Singapore

### ISSUE THREAD

**Issue by vmarkovtsev** [src-d/conferences#161]
**URL**: https://mlconference.ai

**Location**: Singapore

**Dates**: 2020/03/24-26

**Expenses covered**: I can expect the hotel to be covered. The plane tickets are under "?"

**Topic**: ML

**Call for Papers**:

deadline: 2019/10/21
link: https://callforpapers.sandsmedia.com/

**Notes**:

The MLConference series proved to be high quality, it is worth speaking and attending in any city :smile:

**Comment by vmarkovtsev** [comment]
My submission:

### Towards the convenient ML infra

Summary of our experience with constructing, hacking, and maintaining the machine learning infrastructure for our R&D team of 10 people. What's the difference between a research ML environment and a production ML environment. Evolution of our workflow management. Encouraging the transparent and collaborative ML development culture. What's better, clouds or homebrew. Constrained resources, Docker, Kubernetes, and you. How we solved complex problems, e.g. enabled GPU overcommitting for JupyterHub.

**Comment by bzz** [comment]
> URL: https://mlconference.ai

said it's DECEMBER 9 – 11, 2019 | BERLIN 🤔

**Comment by vmarkovtsev** [comment]
@bzz Yep they haven't updated their web site. The CFP page is fine though.

**Comment by bzz** [comment]
@EgorBu did you end up submitting a workshops proposal as well?

**Comment by EgorBu** [comment]
@bzz , not yet.
